_sass/prism.scss

```diff
@@ -191,3 +191,30 @@ pre[class*="language-"].line-numbers > code {
   padding-right: 0.8em;
   text-align: right;
 }
+
+div.code-toolbar > .toolbar > .toolbar-item > a,
+div.code-toolbar > .toolbar > .toolbar-item > button,
+div.code-toolbar > .toolbar > .toolbar-item > span {
+  color: white;
+  font-size: 0.8em;
+  padding: 0.25em 0.5em;
+  background: #f5f2f0;
+  background: black;
+  /* box-shadow: 0 2px 0 0 rgba(0, 0, 0, 0.2); */
```

Comment:
[Minor] Since this is commented out, do you remind removing it?

Reply:
commend removed in 254f1e3444285ab504d3abbccbc2897852b164a4